REQUIRE

```diff
@@ -1,2 +1,2 @@
-julia 0.6
-Compat 0.62.0
+julia 0.7
+Compat 0.54.0
```

Comment:
Can we just remove compat then?

Reply:
Yes, unless there is some other compat function or macro in the previous versions (I didn't check that).

Reply:
I think this has to be `julia 0.7-` to show that it is for a prerelease of 0.7.